RS: Enable rack-zone awareness for existing clusters & databases (#2493)
RedisLabs/redislabs-docs: content/rs/clusters/configure/rack-zone-awareness.md

1. In **node configuration** > **Cluster configuration**, select **Enable rack-zone awareness**.

1. Enter a **Rack-zone ID** for the current node.

1. Finish [cluster setup]({{<relref "/rs/clusters/new-cluster-setup">}}).

1. For every [node you add to the cluster]({{<relref "/rs/clusters/add-node">}}), assign a different **Rack-zone ID** when prompted.

### Existing cluster

If you did not configure rack-zone awareness during cluster creation, you can configure rack-zone awareness for existing clusters using the [REST API]({{<relref "/rs/references/rest-api">}}):

1. For each node in the cluster, assign a different rack-zone ID using the REST API to [update the node]({{<relref "/rs/references/rest-api/requests/nodes#put-node">}}):

    ```sh
    PUT /v1/nodes/<node-ID>
    { "rack_id": "rack-zone-ID" }
    ```

1. [Update the cluster policy]({{<relref "/rs/references/rest-api/requests/cluster/policy#put-cluster-policy">}}) to enable rack-zone awareness:

    ```sh
    PUT /v1/cluster/policy
    { "rack_aware": true }
    ```

## Enable database rack-zone awareness

> Before you can enable rack-zone awareness for a database, you must [set up rack-zone awareness](#set-up-rack-zone-awareness) for the cluster and its nodes.

Reviewer: ```suggestion
Before you can enable rack-zone awareness for a database, you must configure rack-zone awareness for the cluster and its nodes. For more information, see [set up rack-zone awareness](#set-up-rack-zone-awareness).
```

> You can use the admin console to enable rack-zone awareness for a database:

Reviewer: ```suggestion
To enable rack-zone awareness for a database, in the admin console:
```

> 1. From **create database** (for [new databases]({{<relref "/rs/databases/create">}})) or **database** > **configuration** (for existing databases), make sure **Replication** is enabled.

Reviewer: ```suggestion
1. Check that **Replication** is enabled in **database > configuration** for existing databases. To create a new database, go to **create database** and use instructions in [Create a Redis Enterprise Software database](({{<relref "/rs/databases/create">}})).
```

1. Select **Rack-zone awareness**.

1. **Activate** or **Update** your database.

> 1. For existing databases, you should [rearrange the database shards](#rearrange-database-shards) to ensure your database is optimized for rack-zone awareness.

Reviewer: ```suggestion
1. Rearrange database shards to optimize your database for rack-zone awareness. See how in [rearrange the database shards](#rearrange-database-shards).
```
